config/locales/de/exercise.yml

```
@@ -37,6 +37,8 @@ de:
     download_file_tree:
       file_root: Erstellte Dateien
       gone: Die angeforderte Datei konnte nicht abgerufen werden. Erstellte Dateien werden nur kurzzeitig vorgehalten und dann gelöscht. Bitte führen Sie den Code erneut aus und versuchen Sie dann wieder den Download der Datei.
+    download_proforma:
+      label: Download Proforma Zip
```

Based on our wording in CodeHarbor, I would recommend using ZIP with capital letters.

```suggestion
      label: Download Proforma ZIP
```

In CodeHarbor, we also used the term "ProformaXML". However, this might be too long:

https://github.com/openHPI/codeharbor/blob/432e12f54365293137b439832908999dfff9985d/config/locales/de/proforma_errors.yml#L4

Alternatively, we dropped "ProformaXML" completely 🙈:

https://github.com/openHPI/codeharbor/blob/432e12f54365293137b439832908999dfff9985d/config/locales/de/common.yml#L14

<hr>

Long story short: I would, at least, capitalize ZIP. On everything else, I am undecided.